### ISSUE TITLE
Errata: Clarify ordering of accumulate operations with origin == target

### ISSUE THREAD

**Issue by devreal** [mpi-forum/mpi-issues#144]
# Problem
In §11.7.2 of the MPI standard, the text states:

```
MPI specifies ordering between accumulate operations from one process to the same (or overlapping) memory locations at another process on a per-datatype granularity.
```

This can be read to not include operations with from one process to itself (`origin == target`) [1]. However, there is no reason to exclude operations to `self` from the ordering guarantees (and it seems to be unintentional [2]).

# Proposal
By consistently using the terminology of `origin` and `target` established in the RMA chapter this ambiguity can be avoided:

```
MPI specifies ordering between accumulate operations
from an origin process to the same (or overlapping) memory locations at
a target process on a per-datatype granularity.
```

This proposal fixes another instance where `source` is used instead of `origin`.

# Changes to the Text

See above. 

PR: https://github.com/mpi-forum/mpi-standard/pull/127

# Impact on Implementations

None

# Impact on Users

It is made clear that accumulate operations from a process to itself are ordered according to the ordering rules outlined in §11.7.2.

# References
[1] Merriam Webster: https://www.merriam-webster.com/dictionary/another

> *Definition of another*
> 1 : different or distinct from the one first considered 

[2] RMA WG issue: https://github.com/mpiwg-rma/rma-issues/issues/11

**Comment by wesbland** [comment]
This passed the first and only vote on in Zurich, Switzerland on 2019-09-07.

**Comment by devreal** [comment]
This has been fixed with #127, closing.
